doc/performance-investigations/0010-web-server-worker-class/summary.md

```diff
@@ -0,0 +1,62 @@
+# Web server worker class: Async vs Threads vs Threads + GRPC
+
+We have only ever run the application with Gunicorn in async mode (workers). We should try threads if it is more performant, reliable or consistent. 
```

Comment:
Past doesn't really mean much or provide a justification for a test, specially if you are reading in the future, as context is lost . This is simply a Gunicorn in async mode compared to it using threads with and without gRPC

Reply:
Just following our past pattern. But have updated.